<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAppointmentDialog</class>
 <widget class="QDialog" name="AddAppointmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Записи - Добавление</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayoutAppointmentForm">
     <item row="4" column="0">
      <widget class="QLabel" name="labelAppointmentTime">
       <property name="text">
        <string>Время приёма</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelAppointmentCost">
       <property name="text">
        <string>Стоимость</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPatientPhone">
       <property name="text">
        <string>Номер пациента</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="spinBoxAppointmentCost">
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelAppointmentDate">
       <property name="text">
        <string>Дата приёма</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <layout class="QHBoxLayout" name="horizontalAppointmentDate">
       <item>
        <widget class="QSpinBox" name="spinBoxAppointmentDateDay">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>31</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxAppointmentDateMonth">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxAppointmentDateYear">
         <property name="minimum">
          <number>2012</number>
         </property>
         <property name="maximum">
          <number>2023</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="2">
      <layout class="QHBoxLayout" name="horizontalLayoutAppointmentTime">
       <item>
        <widget class="QComboBox" name="comboBoxAppointmentTimeHour"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxAppointmentTimeMinute"/>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditPatientPhone"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelDoctorPhone">
       <property name="text">
        <string>Номер врача</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEditDoctorPhone"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAppointmentAction">
     <item>
      <widget class="QPushButton" name="pushButtonAppointmentActionAdd">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAppointmentActionCancel">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
